File: AutoFun/app/src/main/java/com/myappcompany/imran/autofun/MainActivity.java
package com.myappcompany.imran.autofun;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.ArrayAdapter;
import android.widget.AutoCompleteTextView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        AutoCompleteTextView autoCompleteTextView = findViewById(R.id.autoCompleteTextView);

        String[] fiends = {"Nick", "Rick", "Sara", "James", "Nina", "Noel", "Nire"};

        ArrayAdapter<String> arrayAdapter = new ArrayAdapter<>(this, android.R.layout.simple_list_item_1, fiends);

        autoCompleteTextView.setAdapter(arrayAdapter);

        autoCompleteTextView.setThreshold(1);

    }
}

File: JavaFaithful/app/src/main/java/com/myappcompany/imran/javafaithful/MainActivity.kt
package com.myappcompany.imran.javafaithful

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val myDawg = Dog("Nick", 5)
    }
}